purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -32,6 +34,7 @@ open class PurchasesConfiguration(builder: Builder) {
         @set:JvmSynthetic @get:JvmSynthetic internal var observerMode: Boolean = false
         @set:JvmSynthetic @get:JvmSynthetic internal var service: ExecutorService? = null
         @set:JvmSynthetic @get:JvmSynthetic internal var store: Store = Store.PLAY_STORE
+        @set:JvmSynthetic @get:JvmSynthetic internal var diagnosticsEnabled: Boolean = false
```

Comment:
We recently decided that we would do better for our customers by leaving it disabled by default. It would certainly cause us to gather very little to no data at the beginning, but our plan is to earn people enabling this by providing extra value to people that enable this. CC @aboedo